api/src/main/java/uk/gov/hmcts/payment/api/dto/BulkScanPaymentRequest.java

```diff
@@ -23,13 +19,14 @@
 
 import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
 
+@With
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 @JsonInclude(NON_NULL)
-@Data
```

Comment:
@Data should have implicit @Getter, @Setter, @ToString, @EqualsAndHashCode and @RequiredArgsConstructor annotations on the class....https://projectlombok.org/features/Data
What is the reason we overriding the default behaviour & explicitly using @Getter & @Setter Annotations.

Reply:
Hi @AyanDuttaHMCTS  pl can u refer this : https://stackoverflow.com/questions/44584487/improve-lombok-data-code-coverage